Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -198,6 +203,7 @@
                                 `${this.paymentFrequencyPropertyName}($select=Id),` +
                                 `${this.optionExtendPropertyName}($select=Id),` +
                                 `${this.terminationDeadlinePropertyName}($select=Id),` +
+                                "AssociatedSystemUsages($select=ItSystemUsage;$expand=ItSystemUsage($select=Uuid))," +
```

Comment:
duplicated select.. already defined in 193.. Also the uuid is the wrong one. It comes from the ItSystemUsage.ItSystem.Uuid so a missin expand